src/modules/LoggedIn/Project/ProjectList.tsx

```diff
@@ -10,7 +20,32 @@ const ProjectList = () => {
         {projectSync.projects.map((project) => {
           return (
             <Col span={6} key={project._id}>
-              <Card hoverable>
+              <Card hoverable
+                // style={{ width: 300 }}
+                // cover={
+                //   <img
+                //     alt="example"
+                //     src="https://gw.alipayobjects.com/zos/rmsportal/JiqGstEfoWAOHiTxclqi.png"
+                //   />
+                // }
+                actions={[
+                  <Popconfirm
+                    title="Are you sure delete this task ?"
```

Comment:
kata katanya diganti project ya, jangan task

```suggestion
                    title="Are you sure delete this project ?"
```